custom_components/rivian/coordinator.py

```diff
@@ -189,7 +198,19 @@ def _build_vehicle_info_dict(self, vijson: dict[str, Any]) -> dict[str, Any]:
             if v
         }
 
-        _LOGGER.debug("Vehicle %s updated: %s", self.vehicle_id, items)
+        if items:
+            _LOGGER.debug("Vehicle %s updated: %s", self.vehicle_id, redact(items))
+
+        if power_state := items.get("powerState"):
+            if power_state.get("value") == "sleep":
+                self._awake.clear()
+            else:
+                self._awake.set()
```

Comment:
Delete this section since it's related to vehicle control and you don't have it initialized anywhere

Reply:
yes, though I was initializing it - never using it correct. Got over zealous in copy/paste not reading since a child screamed "daddy, I need <something random to not go to sleep inserted here>"